builtin/providers/aws/resource_aws_sns_topic_seeker.go

```diff
@@ -0,0 +1,104 @@
+package aws
```

Comment:
I'm not sure this file should be actually called `resource_` since it's not really a resource with its own schema.